src/main/java/com/cachinginapp/enterprise/CachingInController.java

```diff
@@ -13,6 +13,15 @@
 
 import java.util.List;
 
+/**
+ * The controller for CachingInApp REST endpoints and web UI
+ * <p>
+ * This class handles the CRUD operations for CachingInApp "caches", via HTTP actions.
+ * </p>
+ * <p>
+ * This class also serves HTML based web pages, for UI interactions with geo-cache objects.
+ * </p>
+ */
```

Comment:
reimplemented java doc from #24

Reply:
Good, descriptive, and follows proper format.